browser/src/Editor/NeovimEditor.tsx

```diff
@@ -256,6 +262,12 @@ export class NeovimEditor implements IEditor {
                 }
             }
 
+            if (key === "<C-v>" && this._screen.mode === "insert") {
```

Comment:
should we make sure they have the config enabled first?
This would override my mapping I have in vim even if I say
```ts
"editor.clipboard.enabled": false,
```